content/2021-02-16-newsletter-26.md

```diff
@@ -6,20 +6,6 @@ in_search_index = true
 template = "page.html"
 +++
 
-This is the 26th newsletter of the [Embedded WG] where we highlight new progress, celebrate cool projects, thank the community, and advertise projects that need help!
```

Comment:
Maybe keep this line above the fold?

Reply:
Eh, its just such boilerplate. The title says the number. In fact I snuck the numbering out of the template so it doesnt even say what number it is anymore.